src/evergreen/api.py

```diff
@@ -799,6 +801,71 @@ def manifest_for_task(self, task_id: str) -> Manifest:
         url = self._create_url(f"/tasks/{task_id}/manifest")
         return Manifest(self._call_api(url).json(), self)  # type: ignore[arg-type]
 
+    def get_task_annotation(
+        self,
+        task_id: str,
+        execution: Optional[int] = None,
+        fetch_all_executions: Optional[bool] = None,
+    ) -> List[TaskAnnotation]:
+        """
+        Get the task annotations for the given task.
+
+        :param task_id: Id of task to query.
+        :param execution: Execution number of task to query (defaults to 0).
```

Comment:
For us, the default isn't 0, it's just the latest execution.
In this function execution isn't assigned a default, but in annotate_task execution is explicitly set to 0 if not assigned, so it's worth noting that if you were to call "get_task_annotation" with a taskId and no execution, and then "annotate_task" with the same taskId and no execution, you may be updating a different annotation than the one you looked at previously.

Reply:
I'm not sure what I was thinking here. I've updated the phrasing and code.